app/routes/dataclasses/overview.tsx

```diff
@@ -49,18 +40,7 @@ export default function Index() {
       }}
       onSearchChange={setSearch}
       isPending={isPending}
-      overviewButton={
-        <Button
-          icon={IvyIcons.Plus}
-          size='large'
-          variant='primary'
-          type='button'
-          onClick={() => open({ create, exists, type: 'Data Class', namespaceRequired: true })}
-          style={{ height: '32px' }}
-        >
-          {t('dataclasses.newDataclass')}
-        </Button>
-      }
+      control={NewDataClassButton()}
```

Comment:
JSX "functions" should not be called like this. If you do this react cannot do internal optimizations.

```suggestion
      control={<NewDataClassButton />}
```